lesson3/task2_1/task2_1.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "fmt"
+
+func max(s []string) string {
```

Comment:
Hey! I've just wanted to ask You to be consistent with function namings. What this function should do? I cannot answer... Maybe better to name it like "getLongestWord" or smth. more suitable?

Reply:
Ok! understood!